src/test/java/pingis/entities/UserTest.java

```diff
@@ -28,9 +28,9 @@ public void testUserName() {
   }
 
   @Test
-  public void testUserLevel() {
+  public void testUserPoints() {
     final double level = 0.001;
```

Comment:
Why is this named `level` when it's used as the minimum delta for float-point comparisons?